src/sentry/ingest/billing_metrics_consumer.py

```diff
@@ -174,6 +175,11 @@ def _flag_metric_received_for_project(self, generic_metric: GenericMetric) -> No
 
             project = Project.objects.get_from_cache(id=project_id)
             if not project.flags.has_custom_metrics:
+                with sentry_sdk.push_scope() as scope:
+                    scope.set_tag("organization_id", org_id)
+                    scope.set_tag("project_id", project_id)
+                    sentry_sdk.capture_message("A new project has sent the first custom metric")
```

Comment:
Can you add the project slug into the message? That way we can quickly see this in the notification. Maybe send a custom fingerprint along so it groups well.

Reply:
I did not send it exactly because of the fingerprint, I didn't know we could set a custom one. Will do

Reply:
```python
capture_message("foo", fingerprint=['my-shit-here'])
```

should work